src/components/PropertyGroupDropdown.tsx

```diff
@@ -7,21 +9,25 @@ type DropdownProps = {
 
 export function PropertyGroupDropdown (props: DropdownProps) {
     const possiblePropertyGroups = [
-        'All',
-        'Retail',
-        'Industrial',
-        'Commercial'
+        { value: 'All', label: 'All' },
+        { value: 'Retail', label: 'Retail' },
+        { value: 'Industrial', label: 'Industrial' },
+        { value: 'Commercial', label: 'Commercial' }
     ]
 
     return (
-        <label className="six dropdown">Property Group (Portals) <span className='downArrow'> ⬇ </span>
-            <select multiple value={props.chosenPropertyGroups} className="dropdown-content" id="#propertyGroup" onChange={(e: ChangeEvent<HTMLSelectElement>) => {
-                props.changePropertyGroups([...e.target.selectedOptions].map((option) => option.value))
-            }} tabIndex={3} >
-                {possiblePropertyGroups.map((propertyGroup) => {
-                    return <option key={propertyGroup} value={propertyGroup}>{propertyGroup}</option>
-                })}
-            </select>
+        <label className="six">Property Group (Portals)
+            <Select
+                getValue={props.chosenPropertyGroups}
+                id="#propertyGroup"
+                isMulti
+                onChange={(e: PropertyTypes[]) => {
```

Comment:
perhaps `newPropertyTypes` instead of e. e is a bit misleading as it is usually used for a UI event or an error.

Reply:
Also the type annotation seems incorrect as this is for groups not types. Perhaps the type should be called ReactSelectOption